client/webpack.rails.config.js

```diff
@@ -1,31 +1,41 @@
 // Run like this:
-// cd client && $(npm bin)/webpack -w --config webpack.rails.config.js
+// cd client && npm run build:dev
 // Note that Foreman (Procfile.dev) has also been configured to take care of this.
 
 // NOTE: All style sheets handled by the asset pipeline in rails
 
+const webpack = require('webpack');
 const config = require('./webpack.common.config');
 
+const devBuild = process.env.NODE_ENV !== 'production';
+
 config.output = {
-  filename: 'client-bundle.js',
+  filename: '[name].js',
   path: '../app/assets/javascripts/generated',
 };
 
 // You can add entry points specific to rails here
-config.entry.push('./scripts/rails_only');
+config.entry.vendor.unshift('./scripts/rails_only');
```

Comment:
There are polyfills there, I apply them before anything else. But I think they should be moved to `vendor` bundle.